tests/phpunit/tests/rest-api/rest-application-passwords-controller.php

```diff
@@ -69,6 +69,12 @@ public function setUp() {
 		add_filter( 'wp_is_application_passwords_available', '__return_true' );
 	}
 
+	public function tearDown() {
+		parent::tearDown();
+
+		unset( $_SERVER['PHP_AUTH_USER'], $_SERVER['PHP_AUTH_PW'], $GLOBALS['wp_rest_application_password_status'], $GLOBALS['wp_rest_application_password_uuid'] );
```

Comment:
```suggestion
		unset( $_SERVER['PHP_AUTH_USER'], $_SERVER['PHP_AUTH_PW'], $GLOBALS['wp_rest_application_password_status'], $GLOBALS['wp_rest_application_password_uuid'] );
		parent::tearDown();
```

Nit: This matches how it's done in _most_ tests, but it's a bit of a mix within core.